.gitlab/ci/.gitlab-ci.global.yml

```diff
@@ -246,6 +246,12 @@
   - |
   - section_end "Checkout upload commit content nightly"
 
+.fetch-through-merge-base: &fetch-through-merge-base
+  - section_start "Fetch Through Merge Base"
+  - chmod +x ./Tests/scripts/fetch_through_merge_base.sh
```

Comment:
please run this locally and commit, and you can delete this file after the commit